google/ggrc-core
# QUICK-FIX Remove useless bootstrap.py

# Issue description

Code cleanup.

# Steps to test the changes

Check the output of `alembic ggrc revision --autogenerate`, no VARCHAR lengths should change.

# Solution description

The old TODO mark to check the default string length for MySQL was correct: MySQL uses `VARCHAR(250)` by default. We don't need this override while we're on MySQL, and we'll need to address the incompatibilities once we decide to migrate from MySQL to anything else.

# Sanity checklist




# PR Review checklist
